src/Impostor.Server/Net/Inner/Objects/Components/InnerCustomNetworkTransform.Api.cs

```diff
@@ -1,8 +1,17 @@
-﻿using Impostor.Api.Net.Inner.Objects.Components;
+using System.Numerics;
+using System.Threading.Tasks;
+using Impostor.Api.Net.Inner.Objects.Components;
 
 namespace Impostor.Server.Net.Inner.Objects.Components
 {
     internal partial class InnerCustomNetworkTransform : IInnerCustomNetworkTransform
     {
+        public async ValueTask SnapTo(Vector2 position)
+        {
+            var writer = _game.StartRpc(NetId, RpcCalls.SnapTo);
```

Comment:
Update the `_lastSequenceId` of the InnerCustomNetworkTransform before sending the RPC.